### PR TITLE
Fix DG

### PR DESCRIPTION
**Pull request by Joshy837**
Fix spelling n wording

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1379,169 +1383,175 @@ testers are expected to do more *exploratory* testing.
...
-  2. Test case: `/note ; name : Poochie ; note : get kibble ; deadline : 2020-10-10`<br>
-     Expected: Woof! Added note to Pooch Contact Supplier PetCo successfully! 🐶
+  
+   1. Prerequisites: The contact to add a note to must exist. This contact can be of `Person`/`Supplier`/`Staff`/`Maintainer` type. You can run the following command to add a note to a contact:
+      ```
+      /add-person ; name : Poochie ; phone : 98883888 ; address : Pooch Street 32 ; email : impoochie@gmail.com
+      ```
+        
+   2. Test case: `/note ; name : Poochie ; note : get kibble ; deadline : 2020-10-10`<br>
+      Expected: Woof! Added note to Pooch Contact Supplier PetCo successfully! 🐶
```
Comment (yleeyilin) [inline docs/DeveloperGuide.md:1455]: name is Poochie, should not be PetCo

**Review by yleeyilin — COMMENTED** [review]
one minor comment to be fixed

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Joshy837) [reply docs/DeveloperGuide.md]: Resolved

**Review by chiageng — APPROVED** [review]
LGTM

**Review by yleeyilin — APPROVED** [review]
lgtm

**Pull request merged**